### DISCUSSION TITLE
Graceful Shutdown in Docker

### DISCUSSION THREAD

**Discussion by wabragg** [AdguardTeam/AdGuardHome#3511] [General]
Does anyone know if SIGTERM or SIGQUIT is the monitored shutdown signal for this container in Docker?

The Docker environment defaults to sending SIGTERM but I would like to use what's appropriate.

**Comment by ainar-g** [comment]
Hello. `SIGINT`, `SIGQUIT`, and `SIGTERM` should all be supported.
